src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -44,6 +49,14 @@ export const SettingsPane: FunctionComponent<{ explorer: Explorer }> = ({
       ? Constants.Queries.UnlimitedPageOption
       : Constants.Queries.CustomPageOption,
   );
+
+  const [enableDataPlaneRBACOption, setEnableDataPlaneRBACOption] = useState<string>(
+    LocalStorageUtility.hasItem(StorageKey.DataPlaneRbacEnabled)
```

Comment:
Does this caching work if I enable RBAC and disable later and re-enable again, etc?

Reply:
Yes, it did work as per my testing.